sigs.yml

```
@@ -696,3 +696,25 @@
     gcLiaison:
       - name: 'Juraci Paixão Kröhling'
         github: jpkrohling
+  - name: Developer Experience
+    meeting: Every Wednesday 9:00 and 18:00 UTC
+    notes:
+      type: gDoc
+      value: 1QDZYAvGJbqgodJaTxRPmRZh-fEJjb6XstvPUst6h50w
+    chat:
+      - type: slack
+        name: '#otel-devex'
+        id: C01S42U83B2
+    invites: calendar-developer-experience
+    sponsors:
+      - name: 'Austin Parker'
+        github: austinlparker
+      - name: 'Ted Young'
+        github: tedsuo
+      - name: 'Liudmila Molkova'
+        github: lmolkova
+    gcLiaison:
+      - name: 'Austin Parker'
+        github: austinlparker
+      - name: 'Ted Young'
+        github: tedsuo
```

```suggestion
    gcLiaison:
      - name: 'Austin Parker'
        github: austinlparker
      - name: 'Ted Young'
        github: tedsuo
```

nit: only one of them should be the official liaison

---

Ah. It looks like @tedsuo is already responsible as liaison for more SIGs than @austinlparker, so to not overload him I'll make Austin the official liaison. But they can tell me otherwise and I'll correct it.